src/components/Badge/Badge.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+    <span :class="classes">
+        {{ text }}
+    </span>
+</template>
+
+<script>
+export default {
+  name: 'Badge',
+  props: {
+    text: {
+      type: String,
+      required: true
+    },
+    variant: {
+      type: String,
+      default: 'primary',
+      validator:
+        value => [
+          'primary',
+          'secondary',
+          'success',
+          'error',
+          'warning',
+          'danger',
+          'light',
+          'dark'
+        ].includes(value)
+    }
+  },
+
+  computed: {
+    classes () {
+      return {
+        badge: true,
+        [`badge--${this.variant}`]: true
+      }
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+    .badge {
```

Comment:
Like the button, use a capital letter for class names :) it's just a convention

Reply:
Done